libmozdata/patchanalysis.py

```diff
@@ -551,6 +552,10 @@ def attachmenthandler(attachments, bugid, data):
         if info['uplift_comment'] and info['uplift_author']:
             break
 
+    # Render comment as html
+    if info['uplift_comment'] and info['uplift_comment']['text']:
+        info['uplift_comment']['html'] = parse_uplift_comment(info['uplift_comment']['text'], Bugzilla.URL, bug['id'])
```

Comment:
I'd keep this a separate function and not call it automatically in `uplift_info` (because `uplift_info` is also used for analyzing large sets of patches, so we don't want to slow it down by adding information that is not going to be useful for the analyses).
So a caller would do something like:
```
info = patchanalysis.uplift_info(123456, 'release')
comment_html = patchanalysis.parse_uplift_comment(info['uplift_comment']['text'], 123456)
```